extruct/rdfa.py

```diff
@@ -23,10 +23,15 @@
 
 initial_context["http://www.w3.org/2011/rdfa-context/rdfa-1.1"].ns.update({
     "twitter": "https://dev.twitter.com/cards#",
-    "fb": "http://ogp.me/ns/fb#"
+    "fb": "http://ogp.me/ns/fb#",
+    'og': 'http://ogp.me/ns#',
+	'music': 'http://ogp.me/ns/music#',
+	'video': 'http://ogp.me/ns/video#',
+	'article': 'http://ogp.me/ns/article#',
+	'book': 'http://ogp.me/ns/book#',
+	'profile': 'http://ogp.me/ns/profile#'
 })
 
-
```

Comment:
A minor cosmetic request: could you please revert unrelated changes (removed lines) here and below - e.g. here two line lines between top-level statements are used, following pep8.

Reply:
@lopuhin I'm not too familiar with pep8 but I've made the changes to the best of my understanding. Please do let me know if you find anything else out of order :)

Reply:
Looks great, thanks @ShivinDass ! By pep-8 I meant https://www.python.org/dev/peps/pep-0008/ but here I mostly wanted to ensure that PR has formatting consistent with current code and no unrelated changes, thank you!